web-ts/src/components/Layout.tsx

```diff
@@ -112,31 +112,31 @@ type PropsType = {
  * If a footer is supplied and there's not enough content (in header and children) to fill the viewbox, the footer will
  * always stick to the bottom of the viewbox.
  */
-class Layout extends React.PureComponent<PropsType> {
-  static defaultProps = {
-    asideStickyTop: 0,
-    darkMode: false
-  }
+const Layout = ({ asideStickyTop, footer, header, toolbar, modal, children }: PropsType) => {
+  const platform = useContext(PlatformContext)
+  const modalVisible = !!modal
+  return (
+    <FlexWrapper>
+      <RichLayout>
+        <div aria-hidden={modalVisible}>
+          {header}
+          <Body>
+            <Aside asideStickyTop={asideStickyTop} platform={platform}>
+              {toolbar}
+            </Aside>
+            <Main>{children}</Main>
+          </Body>
+        </div>
+        {modal}
+        <div aria-hidden={modalVisible}>{footer}</div>
+      </RichLayout>
+    </FlexWrapper>
+  )
+}
 
-  render() {
-    const { asideStickyTop, footer, header, toolbar, modal, children } = this.props
-    const modalVisible = !!modal
-    return (
-      <FlexWrapper>
-        <RichLayout>
-          <div aria-hidden={modalVisible}>
-            {header}
-            <Body>
-              <Aside style={{ top: `${asideStickyTop}px` }}>{toolbar}</Aside>
-              <Main>{children}</Main>
-            </Body>
-          </div>
-          {modal}
-          <div aria-hidden={modalVisible}>{footer}</div>
-        </RichLayout>
-      </FlexWrapper>
-    )
-  }
+Layout.defaultProps = {
+  asideStickyTop: 0,
+  darkMode: false
```

Comment:
Can't we just use the following syntax above:
```js
const Layout = ({ asideStickyTop = 0, footer, header, toolbar, modal, children }: PropsType) => {
```
`Did remove darkMode` in #305, so we will probably get merge conflicts here.